src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -218,14 +215,48 @@ class CallOpcodesSpec extends WordSpec with Matchers {
       }
 
       "return 1" in {
-        call.stateOut.stack.pop._1 shouldEqual UInt256(1)
+        call.stateOut.stack.pop._1 shouldEqual UInt256.One
       }
 
       "consume correct gas (refund call gas, add new account modifier)" in {
         val expectedGas = G_call + G_callvalue + G_newaccount - G_callstipend + fxt.expectedMemCost
         call.stateOut.gasUsed shouldEqual expectedGas
       }
     }
+
+    "calling a precompiled contract" should {
+      val contractAddress = Address(1) // ECDSA recovery
+      val invalidSignature = ByteString(Array.fill(128)(0.toByte))
+      val world = fxt.worldWithoutExtAccount.saveAccount(contractAddress, Account(balance = 0))
+      val context: PC = fxt.context.copy(world = world)
+      val call = CallResult(op = CALL, context = context, to = contractAddress, inputData = invalidSignature,
+        inOffset = 0, inSize = 128, outOffset = 0, outSize = 128
+      )
+
+      "compute a correct result" in {
+        // For invalid signature the return data should be empty, so the memory should not be modified.
+        // This is more interesting than checking valid signatures which are tested elsewhere
+        val (result, _) = call.stateOut.memory.load(call.outOffset, call.outSize)
+        val expected = invalidSignature
+
+        result shouldEqual expected
+      }
+
+      "return 1" in {
+        call.stateOut.stack.pop._1 shouldEqual UInt256.One
+      }
+
+      "update precompiled contract's balance" in {
+        call.extBalance shouldEqual call.value
+        call.ownBalance shouldEqual fxt.initialBalance - call.value
```

Comment:
if default precompiled contract balance is 1 wei what is correct value for precompiled contract after balance change?